WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostPagerActivity.java

```diff
@@ -167,6 +173,17 @@ protected void onPause() {
     @Override
     public boolean onOptionsItemSelected(final MenuItem item) {
         if (item.getItemId() == android.R.id.home) {
+            if (mIsFeed) {
```

Comment:
Oh, I see what this is doing now. When the user hits the back button while viewing a feed item, this shows the feed preview. I don't think this really works - it's quite unexpected to be returned to a view you didn't start from. I think it would make more sense to simply return to the browser, or barring that show the main activity of WPAndroid.